### PR TITLE
Removes `hammer` and `hammer_plug` deps and their code references

### PR DESCRIPTION
**Pull request by ssaunderss**
Resolves Issue https://github.com/firezone/firezone/issues/1216

### CODE REVIEW

**Comment by AndrewDryga** [comment]
@ssaunderss thank you! Can you please remove mentions of rate limits from the docs? 

This is the best kind of PRs where we just remove stuff :).

File: apps/fz_http/lib/fz_http_web/router.ex [hunk 1]
```diff
@@ -15,9 +15,6 @@ defmodule FzHttpWeb.Router do
     plug :put_root_layout, {FzHttpWeb.LayoutView, :root}
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-
-    # XXX: Make this configurable
-    plug Hammer.Plug, @root_rate_limit
```
Comment (jamilbk) [inline apps/fz_http/lib/fz_http_web/router.ex:20]: I think you need to also remove this attribute:

https://github.com/firezone/firezone/actions/runs/3767295825/jobs/6405527115#step:16:289

**Review by jamilbk — CHANGES_REQUESTED** [review]

**Comment by ssaunderss** [comment]
> @ssaunderss thank you! Can you please remove mentions of rate limits from the docs?

@AndrewDryga I only found one place in the docs that mentioned the in house rate limiter, everything else looks like its nginx related to me. Let me know if I missed anything!

**Review by jamilbk — APPROVED** [review]
Just in time for the upcoming REST API. Thanks @ssaunderss!

**Pull request merged**